samples/app-dev/Connectors/Miscellaneous/ODataS4hana/readme.md

```diff
@@ -0,0 +1,253 @@
+# OData with SAP S4 Hana Example
+
+
+## Description
+
+This example demonstrates how we can create SAP S4 Hana connection using OData connector and use all OData CRUD activities for SAP S4 Hana server.
+OData app with SAP S4 Hana bascially contain 2 activities i.e. OData Query and OData Modify. OData Modify contains three operation i.e. Create, Update and Delete. 
+The main purpose of these activities are:
+a) To fetch SAP S4 Hana data using OData Query activity.
+b) To create data in SAP S4 Hana server using Create operation of OData Modify activity.
+c) To update data in SAP S4 Hana server using Update operation of OData Modify activity.
+d) To delete data in SAP S4 Hana server using Delete operation of OData Modify activity.
+
+## Prerequisites
+
+1. Ensure that SAP S4 Hana services are up and running
+  
+## Import the sample
+
+1. Download the sample json file i.e. ODataS4HanaCRUD.json.
+
+2. Create a new empty app
+
+![Create an app](../../../import-screenshots/ODataS4Hana/60.png)
```

Comment:
you can reuse common screenshots from import-screenshots parent directory
check for other common screenshots as well like for flow tester